Modules/TOF/include/TOF/TaskDigits.h

```diff
@@ -88,28 +92,22 @@ class TaskDigits final : public TaskInterface
   std::shared_ptr<TH1F> mTOFRawsToTIC = nullptr; /// TOF Raws - Hit ToT (ns) - I/C side
   std::shared_ptr<TH1F> mTOFRawsToTOC = nullptr; /// TOF Raws - Hit ToT (ns) - O/C side
 
-  std::shared_ptr<TH1F> mTOFRawsLTMHits = nullptr; /// LTMs OR signals
-  std::shared_ptr<TH2F> mTOFrefMap = nullptr;      /// TOF enabled channel reference map
-  std::shared_ptr<TH2F> mTOFRawHitMap = nullptr;   /// TOF raw hit map (1 bin = 1 FEA = 24 channels)
-
-  std::shared_ptr<TH2I> mTOFDecodingErrors = nullptr; /// Decoding error monitoring
-
-  std::shared_ptr<TH1F> mTOFOrphansTime = nullptr;          /// TOF Raws - Orphans time (ns)
-  std::shared_ptr<TH2F> mTOFRawTimeVsTRM035 = nullptr;      /// TOF raws - Hit time vs TRM - crates 0 to 35
-  std::shared_ptr<TH2F> mTOFRawTimeVsTRM3671 = nullptr;     /// TOF raws - Hit time vs TRM - crates 36 to 72
-  std::shared_ptr<TH2F> mTOFTimeVsStrip = nullptr;          /// TOF raw hit time vs. MRPC (along z axis)
-  std::shared_ptr<TH2F> mTOFtimeVsBCID = nullptr;           /// TOF time vs BCID
-  std::shared_ptr<TH2F> mTOFchannelEfficiencyMap = nullptr; /// TOF channels (HWok && efficient && !noisy && !problematic)
-  std::shared_ptr<TH2F> mTOFhitsCTTM = nullptr;             /// Map of hit pads according to CTTM numbering
-  std::shared_ptr<TH2F> mTOFmacropadCTTM = nullptr;         /// Map of hit macropads according to CTTM numbering
-  std::shared_ptr<TH2F> mTOFmacropadDeltaPhiTime = nullptr; /// #Deltat vs #Delta#Phi of hit macropads
-  std::shared_ptr<TH2I> mBXVsCttmBit = nullptr;             /// BX ID in TOF matching window vs trg channel
-  std::shared_ptr<TH2F> mTimeVsCttmBit = nullptr;           /// TOF raw time vs trg channel
-  std::shared_ptr<TH2F> mTOFRawHitMap24 = nullptr;          /// TOF average raw hits/channel map (1 bin = 1 FEA = 24 channels)
-  std::shared_ptr<TH2I> mHitMultiVsDDL = nullptr;           /// TOF raw hit multiplicity per event vs DDL
-  std::shared_ptr<TH1I> mNfiredMacropad = nullptr;          /// Number of fired TOF macropads per event
-  std::shared_ptr<TProfile2D> mOrbitDDL = nullptr;          /// Orbits per crate
-  std::shared_ptr<TH1I> mROWSize = nullptr;                 ///Readout window size
+  // std::shared_ptr<TH1F> mTOFRawsLTMHits = nullptr;          /// LTMs OR signals
```

Comment:
We usually encourage people to remove their code rather than commenting out. Not very important.

Reply:
Indeed this was more to keep it as a bookmark